app/views/services/select-org-type.njk

```diff
@@ -60,8 +62,36 @@
         html: "<p class=\"govuk-body\">Contact <a class=\"govuk-link\" href=\"mailto:govuk-pay-support@digital.cabinet-office.gov.uk\">govuk-pay-support@digital.cabinet-office.gov.uk</a> for help deciding which provider you should use.</p>"
       }) }}
 
-      {{ govukButton({ text: "Create service" })}}
+      {{ govukButton({
+        html: "Create service <span class=\"spinner\"></span>",
+        id: "submit-button",
+        preventDoubleClick: true
+      }) }}
     </form>
 
+    <div id="spinner-container" hidden="hidden" aria-hidden="true">
+      {{ spinner({
+        text: "We're creating your service, please wait..."
+      }) }}
+    </div>
   </div>
+
+  <script>
```

Comment:
Can we move the JS out of this page and into a JS file please?  It can either:

 - Be a specific JS file that's loaded into this page only
 - Be included in a more general JS bundle and only trigger on this page

Reply:
This should be fixed now